wdl2cwl/main.py

```diff
@@ -149,10 +149,17 @@ def get_time_minutes_requirement(
         return f"$({time_minutes_str} * 60)"
 
     def get_outdir_requirement(
-        self, outdir: Union[WDL.Expr.Get, WDL.Expr.Apply]
-    ) -> int:
+        self, outdir: Union[WDL.Expr.Get, WDL.Expr.Apply, WDL.Expr.String]
+    ) -> Union[int, str]:
         """Produce the memory requirement for the output directory from WDL runtime disks."""
-        # This is yet to be implemented. After Feature Parity.
+        if not outdir.literal:
+            if isinstance(outdir, WDL.Expr.String):
+                for part in outdir.parts:
+                    if isinstance(part, WDL.Expr.Placeholder):
+                        outdir_str = self.get_expr(part)[1:]
+                        return f"$({outdir_str}*1024)"
```

Comment:
What if the WDL `disks` entry has used non-default units like ` disks: "local-disk ${disk} MiB" ?